### PR TITLE
Deprecate Yank protocols

### PR DESCRIPTION
**Pull request by mattwthompson**
```python3
>>> from openff.evaluator.protocols.yank import *
Warning on use of the timeseries module: If the inherent timescales of the system are long compared to those being analyzed, this statistical inefficiency may be an underestimate.  The estimate presumes the use of many statistically independent samples.  Tests should be performed to assess whether this condition is satisfied.   Be cautious in the interpretation of the data.
/Users/mattthompson/software/openff-evaluator/openff/evaluator/protocols/yank.py:62: YankDepcrationWarning: The YANK protocol has been deprecated and will be removed in a future release. Free energy calculations will be enabled by Perses in the future.
  warnings.warn(

### CODE REVIEW

File: openff/evaluator/protocols/yank.py [hunk 1]
```diff
@@ -54,6 +55,17 @@
 logger = logging.getLogger(__name__)
 
 
+class YankDepcrationWarning(Warning):
```
Comment (lilyminium) [inline openff/evaluator/protocols/yank.py:58]:
```suggestion
class YankDeprecrationWarning(Warning):
```

File: openff/evaluator/protocols/yank.py [hunk 2]
```diff
@@ -54,6 +55,17 @@
...
+    pass
+
+
+warnings.warn(
+    "The YANK protocol has been deprecated and will be removed in a future release. "
+    "Free energy calculations will be enabled by Perses in the future.",
+    YankDepcrationWarning,
```
Comment (lilyminium) [inline openff/evaluator/protocols/yank.py:65]:
```suggestion
    YankDeprecrationWarning,
```

**Review by lilyminium — APPROVED** [review]
LGTM! Just a typo suggestion, but I won't block the merge.

File: openff/evaluator/protocols/yank.py (hunk 1 above)
Comment (mattwthompson) [reply openff/evaluator/protocols/yank.py:58]: Oof

**Pull request merged**